development-tools/clinic.rst

```diff
@@ -834,8 +834,7 @@ To save time, and to minimize how much you need to learn
 to achieve your first port to Argument Clinic, the walkthrough above tells
 you to use "legacy converters".  "Legacy converters" are a convenience,
 designed explicitly to make porting existing code to Argument Clinic
-easier.  And to be clear, their use is acceptable when porting code for
-Python 3.4.
+easier.
```

Comment:
Maybe this sentence is useful since it's about legacy converters?  @erlend-aasland?

Reply:
I don't understand this sentence. IIRC, legacy converters are just aliases for "normal" converters. I can take a closer look tomorrow. Ping me again if I take too long.

Reply:
@erlend-aasland ping :)

Reply:
I'm fine with removing this. AFAICS, it has no relevance anymore. If it causes great havoc, I'll take the blame :)